.github/workflows/deploy.yml

```diff
@@ -2,7 +2,7 @@
 name: Deploy Kurtosis CDK
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
do we foresee any risks here? 
`The pull_request_target event grants workflows triggered by pull requests from forks access to repository secrets and a read/write GITHUB_TOKEN. That is inherently risky if the workflow inadvertently exposes these secrets or allows for unauthorized modifications to the repository.`

Reply:
Yes, that's essentially the purpose of this job. Otherwise, we wouldn't be able to use secrets in pull requests triggered by external contributors. This is also why there's an approval step.